app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -302,6 +302,9 @@ def pod_definition
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v1.0.z'
+    fq_inspector_image = @ems_configs.fetch_path(:ems_kubernetes, :image_inspector)
+    return fq_inspector_image unless fq_inspector_image.blank?
+    _log.warn('image_inspector not explicitly configured.')
+    'image-inspector:latest'
```

Comment:
1. this method can be: ```@ems_configs.fetch_path(:ems_kubernetes, :image_inspector)``` If someone messes up the configuration we can burn and crash (like with 'miq_namespace') so no need to test anything. 
2. please squash all your commits, the final patch should have one commit

thanks!